src/schema/v2/artwork/index.ts

```diff
@@ -1124,8 +1124,11 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
         },
       },
       priceListed: {
-        type: GraphQLFloat,
-        resolve: ({ price_listed }) => price_listed,
+        type: Money,
+        resolve: ({ price_listed: price_listed, price_currency: currency }) => {
+          const cents = price_listed * 100
```

Comment:
We can't simply `* 100` to convert an amount from major unit to minor unit because different currencies have different ratio, e.g. USD is 100 and JPY is 1.

Metaphysics maintains its own ratios for common currencies in [currency_codes.json](https://github.com/artsy/metaphysics/blob/98e7b3b7f6dc00932f9614b87f885417d51a4a08/src/lib/currency_codes.json). We can use it for the conversion based on the price currency. An example conversion can be seen [here](https://github.com/artsy/metaphysics/blob/98e7b3b7f6dc00932f9614b87f885417d51a4a08/src/schema/v2/fields/money.ts#L73).